cmd/monaco/download/downloaders.go

```diff
@@ -89,6 +90,10 @@ func removeSkipDownload(api api.API) bool {
 	return false
 }
 
+func ShouldApplyFilter() bool {
```

Comment:
I don't think this should be exported anymore now

Reply:
I agree. To avoid massive changes in tests, this is exported. Is it enough to write a comment?

Reply:
Do the tests ever set the environment variables though?
From what I saw, only `TestDownload_SkipConfigBeforeDownload` does actually set the env vars, all other uses could set the value they need explicitly or just default to true. 

For `TestDownload_SkipConfigBeforeDownload` IMO on the downloader level this should change to just verify the boolean option works as intended - defaulting to true and possible to turn off. 

That the env vars translate to the correct input for the downloader that is created in the cmd package should IMO be tested there, not by tying the logic of cmd and pkg together. 
I think the test could move to the cmd/ tests easily as it just seems to use public methods of the downloader anyway?